.vscode/launch.json

```diff
@@ -268,6 +268,24 @@
 				"group": "2_attach"
 			}
 		},
+		{
+			/* Added for "VS Code Selfhost Test Provider" extension support */
+			"type": "pwa-chrome",
+			"request": "attach",
+			"name": "Attach to VS Code",
```

Comment:
should we update the text to refer to Azure Data Studio, assuming this isn't attaching to VS Code.

Reply:
Unfortunately it seems like the provider is hardcoded to use a config with this name, so unless we fork it and make our own or get them to make a change to allow overriding this I don't think we'll be change this one.

Reply:
oh, yeah, the name is not important enough to fork the code